### PR TITLE
Add advanced use

### PR DESCRIPTION
**Pull request by lnauta**
Adds some advanced examples for spider

### CODE REVIEW

**Comment by lnauta** [comment]
I forgot to merge this in august 😓

**Review by KristenLutz — APPROVED** [review]
wow I can't believe no conflicts have popped up since this was created! I've edited a couple pages since and they just happen to not overlap

**Comment by natalieda** [comment]
There are some content overlaps with this merge request. I think section '13. Advanced Spider commands' should be integrated in the existing section '7. Workflows' as similar topics are addressed in 7.1 and 7.3. Also it is best to remove it from the main tree.

File: source/Pages/advanced_commands.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.. _advanced-usage:
+*****************
+Advanced Spider commands
+*****************
+
+Job dependencies
```
Comment (natalieda) [inline source/Pages/advanced_commands.rst:6]: I suggest to move this in section 7.1.

File: source/Pages/advanced_commands.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.. code-block:: bash
+
+   cat slurm-2560644.out
+   
+   Wed Aug 17 11:27:56 CEST 2022
+   bye!
+ 
+And we see that the first job slept for 30 seconds and the second job waited until the first was finished!
+
+Recurring jobs
```
Comment (natalieda) [inline source/Pages/advanced_commands.rst:83]: I suggest to move this in section 7.3.

File: source/Pages/workflows.rst [hunk 3]
```diff
@@ -145,6 +145,7 @@ If you would like to make changes in your cronjob you need to login directly to
 where it was created (tip: to check which node you are on, you can type the command ``hostname``). This
 may also affect your workflows in case of maintenance on the login node you run your cronjobs. 
 
+For a more sustainable option for automated jobs, see the recurring subsection in the :ref:`Advanced Spider commands section<advanced-usage>`.
```
Comment (natalieda) [inline source/Pages/workflows.rst:148]: Is there a reason to split the two sections?

File: source/Pages/advanced_commands.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+==============
+
+If there is a recurring job you want run on a certain schedule, in principle there is ``cron`` available on Spider. However, cron is tied to a single machine, which in the case of Spider would be either a user-interface (ui) node, or a worker node (wn). If a cron job is setup on one of these machines and this machine is not available, your job will not run. 
+
+For more robust recurring jobs, there is ``scrontab``, a SLURM integration of cron. ``scrontab`` has identical syntax to ``cron``. However, the job you submit is added to the SLURM database and run on the defined schedule when the resources are available. The job will only start once the resources are available, so your job may not always run at the exact same time (unlike ``cron``). Your job will be scheduled to run on a worker node, regardless of where it was submitted. 
+
+Therefore we advice users to use ``scrontab`` instead of ``crontab`` when they want to set up cron jobs. The ``scrontab`` documentation can be found `here <https://slurm.schedmd.com/scrontab.html>`_.
```
[the remark this answers is not in the document]
Comment (lnauta) [reply source/Pages/advanced_commands.rst:90]: It has been tested by RO and confirmed to work properly.

**Pull request merged**